core/src/main/scala/cats/syntax/monadError.scala

```diff
@@ -0,0 +1,16 @@
+package cats
+package syntax
+
+trait MonadErrorSyntax {
+
+  implicit def monadErrorSyntax[F[_], E, A](fa: F[A])(implicit F: MonadError[F, E]): MonadErrorOps[F, E, A] =
+    new MonadErrorOps(fa)
```

Comment:
I noticed that the `applicativeErrorSyntax` method was defined for a `F[_, _]` (notice the kind), but I have type inference issues with it. So here I defined it for a `F[_]`. Am I wrong?